src/clr-angular/data/datagrid/datagrid-column.spec.ts

```diff
@@ -328,7 +334,7 @@ export default function(): void {
         });
 
         describe("View filters", function() {
-            it("doesn't display any filter by default", function() {
+            it("doesn't displayType any filter by default", function() {
```

Comment:
Oops. 😛

Reply:
IntelliJ stopped working for me and I tried to replace a var name in one file with VSCode. Clearly it did not go well. I almost can't believe that this is the only one that I didn't find. 😳

Fixed.